Backpack-SwiftUI/AppSearchModal/Classes/AppSearchModalContentView.swift

```diff
@@ -63,8 +63,15 @@ struct AppSearchModalContentView: View {
     }
     
     struct ItemCell: View {
+        @Environment(\.sizeCategory) var sizeCategory
+
         let item: BPKAppSearchModalContent.Item
         
+        private func subtitleLineLimit() -> Int? {
+            let isDefaultSizeOrSmaller = sizeCategory <= .large
+            return isDefaultSizeOrSmaller ? 2 : nil
+        }
```

Comment:
Can we move the function below the body var please :D

Reply:
Done @gert-janvercauteren 😄✅

I don't have merge access in this repository, so not sure should I ask you or @mcomisso to merge this pull request (I'm not familiar with this repository's work-flow and merge process).